src/config/normalize/index.ts

```diff
@@ -190,6 +190,7 @@ async function normalizedSimpleConfig(
         noExecute: commonOptions.noExecute ?? false,
         noWatch: commonOptions.noWatch ?? true,
         watchFiles: await normalizeWatchFiles(commonOptions),
+        sourcemap: commonOptions.sourcemap ?? false,
```

Comment:
1. default value shoud is `undefined`
2. sourcemap maybe from `config.compilation.sourcemap`